pkg/network/driver_interface.go

```diff
@@ -469,21 +520,28 @@ func (dh *DriverHandle) getStatsForHandle() (map[string]int64, error) {
 		}, nil
 	// A FlowHandle handle returns the flow stats specific to this handle
 	case FlowHandle:
+		if dh.lastNumFlowsMissed < uint64(stats.handle.flow_stats.num_flows_missed_max_exceeded) {
+			log.Warnf("Flows missed due to maximum flow limit. %v", stats.handle.flow_stats.num_flows_missed_max_exceeded)
+		}
+		dh.lastNumFlowsMissed = uint64(stats.handle.flow_stats.num_flows_missed_max_exceeded)
 		return map[string]int64{
-			"read_calls":             int64(stats.handle.handle_stats.read_calls),
-			"read_calls_outstanding": int64(stats.handle.handle_stats.read_calls_outstanding),
-			"read_calls_completed":   int64(stats.handle.handle_stats.read_calls_completed),
-			"read_calls_cancelled":   int64(stats.handle.handle_stats.read_calls_cancelled),
-			"write_calls":            int64(stats.handle.handle_stats.write_calls),
-			"write_bytes":            int64(stats.handle.handle_stats.write_bytes),
-			"ioctl_calls":            int64(stats.handle.handle_stats.ioctl_calls),
-			"packets_observed":       int64(stats.handle.flow_stats.packets_observed),
-			"packets_processed_flow": int64(stats.handle.flow_stats.packets_processed),
-			"open_flows":             int64(stats.handle.flow_stats.open_flows),
-			"total_flows":            int64(stats.handle.flow_stats.total_flows),
-			"num_flow_searches":      int64(stats.handle.flow_stats.num_flow_searches),
-			"num_flow_search_misses": int64(stats.handle.flow_stats.num_flow_search_misses),
-			"num_flow_collisions":    int64(stats.handle.flow_stats.num_flow_collisions),
+			"read_calls":                    int64(stats.handle.handle_stats.read_calls),
+			"read_calls_outstanding":        int64(stats.handle.handle_stats.read_calls_outstanding),
+			"read_calls_completed":          int64(stats.handle.handle_stats.read_calls_completed),
+			"read_calls_cancelled":          int64(stats.handle.handle_stats.read_calls_cancelled),
+			"write_calls":                   int64(stats.handle.handle_stats.write_calls),
+			"write_bytes":                   int64(stats.handle.handle_stats.write_bytes),
+			"ioctl_calls":                   int64(stats.handle.handle_stats.ioctl_calls),
+			"packets_observed":              int64(stats.handle.flow_stats.packets_observed),
+			"packets_processed_flow":        int64(stats.handle.flow_stats.packets_processed),
```

Comment:
Is there a reason this variable is named differently? `packets_processed_flow` vs `packets_processed`?

Reply:
so, that's not part of this change. 
Howver, we have `handle.flow_stats.packets_processed` (if this is a flow handle)
we have `handle.transport_stats.packets_processed` (if this is a transport handle (udp/dns, actually collecting packets)

So there are two fields that the `leaf` is `packets_processed`.  I'd guess that it was done to disambiguate `packets_processed_flow` from `packets_processed_transport` (below, not shown unless you expand out the change).

Reply:
There are several other "leaf" stats with the same names (`read_calls_*`, `write_*`). Since these stats not merged into a single list, I don't see a need to differentiate by name here, since the higher-level grouping does that for us (`driver_total_flow_stats` vs `driver_flow_handle_stats`).

Reply:
I think they are merged into a single list; the expvar stats, which is why they're like this.
Also note that this is how it's been; this was not changed in this PR.  Looks like the diff picked up a formatting change.